x-pack/plugins/watcher/__jest__/client_integration/watch_create_threshold.test.tsx

```diff
@@ -0,0 +1,818 @@
+        });
+
+        test('should not allow negative interval size field', () => {
+          const { form } = testBed;
+
+          form.setInputValue('triggerIntervalSizeInput', '-1');
+
+          expect(form.getErrorsMessages()).toContain('Interval size cannot be a negative number');
+        });
+
+        test('should disable the Create button with invalid fields', () => {
+          const { find } = testBed;
+
+          expect(
+            find('saveWatchButton')
+              .getDOMNode()
+              .getAttribute('disabled')
+          ).toEqual('');
+        });
+
+        test('it should enable the Create button and render additonal content with valid fields', async () => {
+          const { form, find, component, exists } = testBed;
+
+          form.setInputValue('nameInput', 'my_test_watch');
+          find('mockComboBox').simulate('change', [{ label: 'index1', value: 'index1' }]); // Using mocked EuiComboBox
+          form.setInputValue('watchTimeFieldSelect', '@timestamp');
+
+          // @ts-ignore (remove when react 16.9.0 is released)
+          await act(async () => {
+            await nextTick();
+            component.update();
+          });
+
+          expect(
+            find('saveWatchButton')
+              .getDOMNode()
+              .getAttribute('disabled')
+          ).toEqual(null);
+
+          expect(find('watchConditionTitle').text()).toBe('Matching the following condition');
+          expect(exists('watchVisualizationChart')).toBe(true);
+          expect(exists('watchActionsPanel')).toBe(true);
+        });
+      });
+
+      describe('actions', () => {
+        beforeEach(async () => {
+          const { form, find, component } = testBed;
+
+          // Set up valid fields needed for actions component to render
+          form.setInputValue('nameInput', WATCH_NAME);
+          find('mockComboBox').simulate('change', [{ label: 'index1', value: 'index1' }]); // Using mocked EuiComboBox
+          form.setInputValue('watchTimeFieldSelect', WATCH_TIME_FIELD);
+
+          // @ts-ignore (remove when react 16.9.0 is released)
+          await act(async () => {
+            await nextTick();
+            component.update();
+          });
+        });
+
+        test('should simulate a logging action', async () => {
+          const { form, find, actions, exists } = testBed;
+
+          const LOGGING_MESSAGE = 'test log message';
+
+          actions.clickAddActionButton();
+          actions.clickActionLink('logging');
+
+          expect(exists('watchActionAccordion')).toBe(true);
+
+          // First, provide invalid field and verify
+          form.setInputValue('loggingTextInput', '');
+
+          expect(form.getErrorsMessages()).toContain('Log text is required.');
+          expect(
+            find('simulateActionButton')
+              .getDOMNode()
+              .getAttribute('disabled')
+          ).toEqual('');
```

Comment:
Same here? And in other tests below.

Reply:
Thanks for the catch. I believe I updated all of the places where this is used.